### PR TITLE
split of unit types

### PR DESCRIPTION
**Pull request by MG-longshot**
Started the split of unit types to individual modules

### CODE REVIEW

**Comment by MG-longshot** [comment]
sill owls canceling

**Review by jimmyjon711 — APPROVED** [review]

File: extras/AFC_prep.py [hunk 1]
```diff
@@ -115,35 +115,16 @@ def PREP(self, gcmd):
...
-                    logo+='E  )\___/(\n'
-                    logo+='A {(@)v(@)}\n'
-                    logo+='D  {|~~~|}\n'
-                    logo+='Y  {/^^^\}\n'
-                    logo+='!   `m-m`\n'
-                    logo+='  ' + UNIT + '\n'
+                
+                try: unit_type = self.printer.lookup_object('AFC_{}'.format(CUR_HUB.type.replace('_', '')))
+                except:
+                    self.ERROR.AFC_error("{} not supported".format(CUR_HUB.type, False))
```
Comment (jimmyjon711) [inline extras/AFC_prep.py:121]: Oops typed this wrong in my example
```suggestion
                    self.ERROR.AFC_error("{} not supported".format(CUR_HUB.type), False)
```

File: extras/AFC.py [hunk 2]
```diff
@@ -813,7 +813,8 @@ def get_status(self, eventtime):
             str["system"]["extruders"][EXTRUDE]['tool_end_sensor']   = True == CUR_EXTRUDER.tool_end_state   if CUR_EXTRUDER.tool_end   is not None else False
             if CUR_EXTRUDER.buffer_name != None:
                 str["system"]["extruders"][EXTRUDE]['buffer']   = CUR_EXTRUDER.buffer_name
-                str["system"]["extruders"][EXTRUDE]['buffer_status']   = CUR_EXTRUDER.buffer.buffer_status()
+                CUR_BUFFER = self.printer.lookup_object('AFC_buffer ' + CUR_EXTRUDER.buffer_name)
```
Comment (jimmyjon711) [inline extras/AFC.py:816]: You should not need to change these 2 lines as `CUR_EXTRUDER` already holds the object for buffer

Comment (jimmyjon711) [reply extras/AFC.py:816]: Main has these same changes as well

Comment (jimmyjon711) [reply extras/AFC.py:816]: ignore this comment for now

**Review by jimmyjon711 — APPROVED** [review]

**Pull request merged**